lib/dor/models/concerns/publishable.rb

```diff
@@ -39,6 +41,7 @@ def thumb
 
        thumb_image
     end
+    deprecation_deprecate :thumb
```

Comment:
`thumb` is still used in generating public XML here: https://github.com/sul-dlss/dor-services/blob/master/lib/dor/services/public_xml_service.rb#L25

Reply:
originally here when first implemented: https://github.com/sul-dlss/dor-services/pull/237/files#diff-831fbf194eaba823278738aa38837ceaR100 (e